cmd_run.go

```diff
@@ -90,6 +93,12 @@ var RunCommand = &cli.Command{
 			EnvVars: []string{"TIROS_RUN_IPFS_HOST", "TIROS_RUN_KUBO_HOST" /* <- legacy */},
 			Value:   "localhost",
 		},
+		&cli.StringFlag{
+			Name:    "ipfs-api-host",
+			Usage:   "port to reach a Kubo-compatible RPC API",
+			EnvVars: []string{"TIROS_RUN_IPFS_API_HOST"},
+			Value:   "127.0.0.1",
+		},
```

Comment:
Is the `ipfs-api-host` different from the `ipfs-host`? This would mean that the IPFS gateway has a different host than the API, is that the case for helia?

Just curious, I'm totally fine adding this additional flag but if we can get away with just using the `ipfs-host` flag, I would prefer just keeping that. If we indeed need both and `ipfs-api-host` is unset, I would fall back to whatever is set with `ipfs-host`.

Reply:
The difference is kubo has two endpoints, an api endpoint and http gateway endpoint. Helia-http-gateway ( and others might) has only one. 

Youre right that we should probably use the single http host for fallback to version via http. I will update this